java/client/src/main/java/glide/connectors/handlers/MessageHandler.java

```diff
@@ -70,21 +72,21 @@ public void handle(Response response) {
             case PUnsubscribe:
             case SUnsubscribe:
                 // ignore for now
-                // TODO log thru logger https://github.com/aws/glide-for-redis/pull/1422
-                System.out.printf(
-                        "Received push notification of type '%s': %s\n",
-                        pushType,
-                        Arrays.stream(values)
-                                .map(v -> v instanceof Number ? v.toString() : String.format("'%s'", v))
-                                .collect(Collectors.joining(" ")));
+                Logger.log(
+                        Logger.Level.INFO,
+                        "SUnsubscribe notification",
+                        String.format(
+                                "Received push notification of type '%s': %s\n",
```

Comment:
Small nit, becase it is logged same for all 6 cases
```suggestion
                        "subscribe/unsubscribe notification",
                        String.format(
                                "Received push notification of type '%s': %s",
```